packages/headless/src/case-assist.index.ts

```diff
@@ -78,6 +79,8 @@ export type {
 } from './controllers/document-suggestion-list/headless-document-suggestion-list';
 export {buildDocumentSuggestionList} from './controllers/document-suggestion-list/headless-document-suggestion-list';
 
+export {buildCaseAssistInteractiveResult as buildInteractiveResult} from './controllers/document-suggestion-list/case-assist-headless-interactive-result';
```

Comment:
This is a breaking change is it not? We would need to also export them under the current names.